src/migrate.js

```diff
@@ -2,29 +2,25 @@
 
 ( function() {
 
+	var rbadVersions = /^[12]\./;
+
 	// Support: IE9 only
 	// IE9 only creates console object when dev tools are first opened
-	// Also, avoid Function#bind here to simplify PhantomJS usage
-	var log = window.console && window.console.log &&
-		function() {
-			window.console.log.apply( window.console, arguments );
-		},
-		rbadVersions = /^[12]\./;
-
-	if ( !log ) {
+	// IE9 console is a host object, callable but doesn't have .apply()
+	if ( !window.console || !window.console.log ) {
```

Comment:
Right, and even when `window.console` exists in IE9 (when the debugger window is active) it is a "host object" that is callable but doesn't have `.apply()`.